### ISSUE TITLE
Brace parsing in the `display` attribute appears to be broken

### ISSUE THREAD

**Issue by coriolinus** [frozenlib/parse-display#17]
# Example

```rust
// for purposes of this example, this struct is defined in an external crate and cannot be modified.
#[derive(Debug, Default)]
struct Point {
    x: i32,
    y: i32,
}

// therefore, this wrapper exists to add `FromStr` and `Display` implementations.
#[derive(Debug, Default, parse_display::FromStr, parse_display::Display)]
#[display("<{0.x}, {0.y}>")]
#[from_str(default)]
struct PointWrapper(Point);
```

# Expected Behavior

As all braces are matched no error is produced.

# Observed Behavior

```
error: invalid format string: expected `'}'` but string was terminated
 --> src/main.rs:9:61
  |
9 | #[derive(Debug, Default, parse_display::FromStr, parse_display::Display)]
  |                                                         -   ^ expected `'}'` in format string
  |                                                         |
  |                                                         because of this opening brace
  |
  = note: if you intended to print `{`, you can escape it using `{{`
```

Note that the span attached to this error is particularly unhelpful.

**Comment by frozenlib** [comment]
Thanks for the bug report.

It used to output a format string like `{:}` when format_spec was empty, but this was an out-of-spec format string, so it has been modified to generate a format string like `{}` in a77faa1db340908890495746744f95f68a0fc460.

**Comment by coriolinus** [comment]
Thank you! When a release goes out containing that commit, then we can close this.

**Comment by frozenlib** [comment]
I have now published version 0.5.1 which includes that commit.
